bitcoin/test/fuzz/miniscript_random.cpp

```diff
@@ -67,12 +68,11 @@ struct TestData {
             if (i & 1) hash160_preimages[hash] = std::vector<unsigned char>(keydata, keydata + 32);
         }
     }
-};
+} TEST_DATA;
```

Comment:
Actually we can't do that, it's causing a fuzz failure when running other targets which `ECC_Start()` too. I think it's because all targets are linked into a single binary, and that it gets executed twice for those that already call it during their initialization?

Reply:
Apart from the crash, globals will also poison the coverage data, as unrelated test cases will mark the constructor of the global as covered.

Ideally each target runs only its own code. Expensive setup code can be put into the init function, which is called once before the target starts.

Reply:
Hmm. I was thinking of just `ECC_Stop`ing at the end of the constructor (since we only need the context for the GetPubkey() there). But then maybe it's worth just not including this change and stay with my previous hack (which does exactly what Marco says)?

Reply:
Done.